.github/workflows/build-kubespray-image.yaml

```diff
@@ -0,0 +1,47 @@
+---
+name: Build kubespray images
+
+on:
+  push:
+    branches: [master, release-.*]
+  release:
+    branches: [master, release-.*]
+
+env:
+  IMAGE_REPO: "kubespray"
+  IMAGE_NAME: "kubespray"
+  IMAGE_REGISTRY: "hub.k8s.li"
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Define image tags by current git repo tag and branch
+        shell: bash
+        run: |
+          git describe --tags --always | sed 's/^/IMAGE_TAG=/' >> $GITHUB_ENV
+          git branch --show-current | sed 's/master/latest/;s/release-//;s/^/IMAGE_TAG_BY_BRANCH=/' >> $GITHUB_ENV
```

Comment:
I did run this `git branch --show-current` on my ubuntu 18.04 machine, and the option `--show-current` is not supported on the machine becuause the git is old.
This might be only my issue because this job specifies ubuntu 20.04.